### PR TITLE
Minor fix for input Kconfig

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary

- input: Remove prompt string from INPUT_MOUSE 
- usbhost: Fix typo error in Kconfig(MOUSE->INPUT_MOUSE)

## Impact
Minor

## Testing
Pass CI

### CODE REVIEW

**Review by jerpelea — APPROVED** [review]

File: drivers/input/Kconfig [hunk 1]
```diff
@@ -14,7 +14,7 @@ menuconfig INPUT
 if INPUT
 
 config INPUT_MOUSE
-	bool "Enable mouse support"
```
Comment (pkarashchenko) [inline drivers/input/Kconfig:17]: Why the text is removed?

Comment (acassis) [reply drivers/input/Kconfig:17]: INPUT_MOUSE is only a flag to indicate we have mouse support, it is activated by other device drivers that implement some kind of "mouse". Note there is not a mouse.c file or equivalent to be compiled when INPUT_MOUSE is enabled

**Review by acassis — APPROVED** [review]

**Review by pkarashchenko — APPROVED** [review]

File: drivers/input/Kconfig (hunk 1 above)
Comment (davids5) [reply drivers/input/Kconfig:17]: If it is a "hidden" bool shouldn't the help go away also?

Comment (xiaoxiang781216) [reply drivers/input/Kconfig:17]: Even it's a hidden option, the mouse driver dev can get help from it when he add new driver option to Kconfig file.

**Pull request merged**